chapters/curate-maintain/content/reading/README.md

```diff
@@ -0,0 +1,13 @@
+### Content
+
+the central part is content management, around a repository
+
+typically Git & GitHub
+
+versioning, maintaining up-to-date
+
+encouraging contributions, integrating changes
+
+creating issues, monitoring issues and pull requests
+
+licensing, ownership
```

Comment:
I don't know if this is a good idea, but maybe we can put some guidelines on how the merging process should be done. For example, we are now in the process of creating the repo and our work style is rather chaotic: we make PRs, we go back to the drawing board, we integrate unfinished PRs, and then we do a re-organization etc. Also, I am reviewing this PR now and I don't know if I should highlight nits in the use/deliver section or I should just limit myself to just reviewing the structure.

What do you think?

Reply:
You're right. I'll add a mention.